SwiftUIDZ/SwiftUIDZ/StartedView.swift

```diff
@@ -0,0 +1,126 @@
+// StartedView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Стартовый экран
+struct StartedView: View {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let yellowColorName = "pastelYellow"
+        static let orangeColorName = "pastelOrange"
+        static let titleText = "ONLINE FURNITURE STORE"
+        static let wifiImageName = "wifi.slash"
+        static let getStartedButtonTitle = "GET STARTED"
+        static let purpleColorName = "pastelPurple"
+        static let darkPurpleColorName = "pastelDarkPurple"
+        static let fontSize: CGFloat = 34
+        static let scale: CGFloat = 0.1
+        static let imageSize: CGFloat = 200
+        static let oneHundredPadding: CGFloat = 100
+        static let notHaveAccountText = "Don't have an account"
+        static let sixtyPadding: CGFloat = 60
+        static let fivePadding: CGFloat = 5
+        static let signInHereButtonTitle = "Sign in here"
+    }
+
+    // MARK: - Body
+
+    var body: some View {
+        NavigationView {
+            ZStack {
+                backgroundGradientView
+                VStack {
+                    mainTextButtonView
+                    getStartedButtonView
+                    notHaveAccountTextView
+                    signInHereButtonView
+                }
+            }
+            .navigationBarHidden(true)
+        }
+    }
+
+    // MARK: - Private Properties
+
+    @ObservedObject private var viewModel = StartedViewModel()
+
+    private var signInHereButtonView: some View {
```

Comment:
```suggestion
    private var signInHereButtonView: some View {
```
signInHereNavLinkView